js/main.js

```diff
@@ -0,0 +1,182 @@
+      $discs.append($rowDiv);
+      $rowDiv.append($discDiv);
+      $discDiv.append($discName);
+      $discDiv.append($discNumbers);
+    } else if (sortedResponse[i].brand !== sortedResponse[i - 1].brand) {
+      if (sortedResponse[i].brand_slug === 'løft-discs') {
+        sortedResponse[i].brand_slug = 'loft-discs';
+      }
+      const $brandDiv = document.createElement('div');
+      const $brandHeader = document.createElement('h3');
+      const $discDiv = document.createElement('div');
+      const $rowDiv = document.createElement('div');
+      const $discName = document.createElement('p');
+      const $discNumbers = document.createElement('p');
+      $brandHeader.textContent = sortedResponse[i].brand;
+      $discName.textContent = sortedResponse[i].name;
+      $discNumbers.textContent = `${sortedResponse[i].speed}|${sortedResponse[i].glide}|${sortedResponse[i].turn}|${sortedResponse[i].fade}|`;
+      $brandDiv.setAttribute('class', 'brand-row');
+      $discDiv.setAttribute('class', `disc ${sortedResponse[i].brand_slug}`);
+      $rowDiv.setAttribute('class', 'disc-row');
+      $discName.setAttribute('class', 'disc-name');
+      $discNumbers.setAttribute('class', 'flight-numbers');
+      $discs.append($brandDiv);
+      $brandDiv.append($brandHeader);
+      $discs.append($rowDiv);
+      $rowDiv.append($discDiv);
+      $discDiv.append($discName);
+      $discDiv.append($discNumbers);
+    } else {
+      if (sortedResponse[i].brand_slug === 'løft-discs') {
+        sortedResponse[i].brand_slug = 'loft-discs';
+      }
+      const $discDiv = document.createElement('div');
+      const $discName = document.createElement('p');
+      const $discNumbers = document.createElement('p');
+      const $allDiscRows = document.querySelectorAll('.disc-row');
+      const previousRow = $allDiscRows[$allDiscRows.length - 1];
+      $discName.textContent = sortedResponse[i].name;
+      $discNumbers.textContent = `${sortedResponse[i].speed}|${sortedResponse[i].glide}|${sortedResponse[i].turn}|${sortedResponse[i].fade}|`;
+      $discDiv.setAttribute('class', `disc ${sortedResponse[i].brand_slug}`);
+      $discName.setAttribute('class', 'disc-name');
+      $discNumbers.setAttribute('class', 'flight-numbers');
+      previousRow.append($discDiv);
+      $discDiv.append($discName);
+      $discDiv.append($discNumbers);
+    }
+  }
+}
+
+xmlHR.send();
+
+// Event listener disc click to show to flight pattern
+
+$searchPage.addEventListener('click', handleDiscClick);
+const $blurModal = document.querySelector('.background-blur-modal');
+$blurModal.addEventListener('click', removeBlur);
+const $discModal = document.querySelector('.disc-modal');
+const $flightPatternImage = document.querySelector('.flight-pattern-image');
+const $anchor = document.querySelector('a');
+
+async function handleDiscClick(event) {
+  const response = await fetch('https://discit-api.fly.dev/disc');
+  const data = await response.text();
```

Comment:
Until you have a good understanding of promises and how they work, you should use `XMLHttpRequest` instead